spec/dummy/config/application.rb

```diff
@@ -23,9 +23,6 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
-
-    # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
```

Comment:
Yeah I suppose we don't care about opting-into this behavior for Rails < 5.1. But that does make me wonder if we should configure Travis to run against multiple Rails versions if we really care about supporting them. (More effort than it is worth?)